src/main/java/ro/unibuc/triplea/domain/auth/model/entity/Game.java

```diff
@@ -0,0 +1,57 @@
+package ro.unibuc.triplea.domain.auth.model.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+
+@Entity
+@Table(name = "Games")
+@Data
+@EqualsAndHashCode(callSuper = false)
+public class Game extends BaseEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    @Column(name = "game_steam_id", nullable = false)
+    private Integer gameSteamId;
```

Comment:
Why do we need id and gameSteamId. Can't be use gameSteamId as primary key?

Reply:
What if you later decide to extend this to games from other providers?

Reply:
Good question, then I think we can leave this code like this: leave id as the internal identifier for all games in our platform, and add columns for the id from each game provider, as needed

Reply:
Maybe a good approach (if we would have another providers) is to create a base class named Game and create another class SteamGame which will contain gameSteamId

Reply:
Created SteamGame class that extends Game class